x-pack/plugins/observability_solution/observability_ai_assistant_app/public/components/prompt_editor/prompt_editor.tsx

```diff
@@ -49,6 +50,8 @@ export function PromptEditor({
 
   const [hasFocus, setHasFocus] = useState(false);
 
+  const { previousPrompts, addLastUsed: addPrompt } = useLastUsedPrompts();
```

Comment:
What's the point of renaming here?

Reply:
It is the result of your previous request to rename, and VSCode applying the rename symbol.